certs/server.cnf

```diff
@@ -0,0 +1,26 @@
+[ req ]
+default_bits        = 2048
+prompt              = no
+default_md          = sha256
+distinguished_name  = req_distinguished_name
+x509_extensions     = v3_extensions
+
+[ req_distinguished_name ]
+C  = US
+ST = Florida
+L  = Jupiter
+O  = ZeroC,\ Inc.
+OU = Ice
+CN = 127.0.0.1
+emailAddress = info@zeroc.com
+
+[ v3_extensions ]
+subjectKeyIdentifier = hash
+authorityKeyIdentifier = keyid:always,issuer
+keyUsage = critical, digitalSignature, nonRepudiation, keyEncipherment
+extendedKeyUsage = serverAuth, clientAuth
```

Comment:
Is `clientAuth` correct here?

Reply:
We have it before but not sure if it is strictly required.

Reply:
I'm pretty sure it's incorrect, can we try remming it.

Reply:
Seems this is required for C#, when the server certificate is used as the "client".

This is a call to self:

```
    public override async Task<int>
    opWithResultAsyncDispatchAsync(Ice.Current current)
    {
        test(Thread.CurrentThread.Name.Contains("Ice.ThreadPool.Server"));
        await Task.Yield();
        test(!Thread.CurrentThread.Name.Contains("Ice.ThreadPool.Server"));
        return await self(current).opWithResultAsync();
    }
```

I going to keep this for now and add a comment in the file.